### PR TITLE
Migrate ngraph backend test of activation operations

### PR DESCRIPTION
**Pull request by wilson-seok**
### Details:
 - add clamp/elu/exp/gelu/hard_sigmoid/hsigmoid/hswish in template plugin reference test
 - remove clamp/elu/exp/gelu/hard_sigmoid in ngraph backend test

### Tickets:
 - 63946
 - 63955
 - 63957
 - 63961
 - 64111
 - 63962
 - 64007

### CODE REVIEW

File: docs/template_plugin/tests/functional/op_reference/clamp.cpp [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        return std::make_shared<Function>(NodeVector {Clamp}, ParameterVector {in});
+    }
+};
+
+TEST_P(ReferenceClampLayerTest, CompareWithRefs) {
+    Exec();
+}
+
+INSTANTIATE_TEST_SUITE_P(
+    smoke_Clamp_With_Hardcoded_Refs, ReferenceClampLayerTest,
```
Comment (ilyachur) [inline docs/template_plugin/tests/functional/op_reference/clamp.cpp:76]: Please add tests for all supported data types

Comment (wilson-seok) [reply docs/template_plugin/tests/functional/op_reference/clamp.cpp]: Updated. Thanks!

**Comment by wilson-seok** [comment]
cc @geunhwan  @steve-y @davidsnam-intel

File: docs/template_plugin/tests/functional/op_reference/clamp.cpp [hunk 2]
```diff
@@ -0,0 +1,208 @@
+// Copyright (C) 2021 Intel Corporation
```
Comment (iefode) [inline docs/template_plugin/tests/functional/op_reference/clamp.cpp:1]: please remove ngraph backend tests for these ops

Comment (wilson-seok) [reply docs/template_plugin/tests/functional/op_reference/clamp.cpp:1]: updated!

**Review by iefode — APPROVED** [review]

File: docs/template_plugin/tests/functional/op_reference/prelu.cpp [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        result << "oType=" << param.outType << "_";
+        result << "slopeShape=" << param.negativeSlopeShape;
+        return result.str();
+    }
+
+private:
+    static std::shared_ptr<Function> CreateFunction(const PreluParams& params) {
+        const auto in = std::make_shared<op::Parameter>(params.inType, params.pshape);
+        //const auto SLOPE = op::Constant::create(input_type, negativeSlopeShape, negativeSlope->get());
+        const auto SLOPE = std::make_shared<op::Parameter>(params.inType, params.negativeSlopeShape);
```
Comment (wilson-seok) [inline docs/template_plugin/tests/functional/op_reference/prelu.cpp:64]: @ilyachur @iefode PRelu ngraph backend test runs slope as both parameter and constant input. I can implement Parameter input as the code. But I cannot for Constant input.
-Can I extract pointer of vector from Blob::Ptr?
-If not, I can make std::vector<float>  only for fp32 as the ngraph backend test of prelu uses only fp32.

File: docs/template_plugin/tests/functional/op_reference/mish.cpp [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                    input,
+                    expected)
+    };
+    return mishParams;
+}
+
+std::vector<MishParams> generateMishCombinedParams() {
+    const std::vector<std::vector<MishParams>> mishTypeParams {
+        generateMishFloatParams<element::Type_t::f32>({2, 5}),
+        generateMishFloatParams<element::Type_t::f32>({2, 3, 4, 5}),
```
Comment (wilson-seok) [inline docs/template_plugin/tests/functional/op_reference/mish.cpp:100]: @ilyachur @iefode The backend test of Mish uses dynamic dimenstion/shape. But it doesn't run in template plugin reference test. It seems that template plugin may not support dynamic shape?

File: docs/template_plugin/tests/functional/op_reference/log_softmax.cpp [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+                    0),
+        LogSoftmaxParams(ngraph::PartialShape {2, 4},
+                    IN_ET,
+                    std::vector<T>{0, 1, 2, 3, 10000, 10001, 10002, 10003},
+                    std::vector<T>{-10000., -10000., -10000., -10000., 0., 0., 0., 0.},
+                    0),
+        LogSoftmaxParams(ngraph::PartialShape {2, 4},
+                    IN_ET,
+                    std::vector<T>{0, 1, 2, 3, 10000, 10001, 10002, 10003},
+                    std::vector<T>{-3.4401896, -2.4401896, -1.4401897, -0.4401897, -3.4401896, -2.4401896, -1.4401897, -0.4401897},
```
Comment (wilson-seok) [inline docs/template_plugin/tests/functional/op_reference/log_softmax.cpp:90]: @ilyachur @iefode one element(10003) passes in fp32, but fails in fp16. So I removed fp16 in test. Do I need to create JIRA ticket for fp16 accuracy issue so  template plugin owner make it fixed? I found that adding in skip_tests_config.cpp doesn't affect to op reference test cases.

**Comment by wilson-seok** [comment]
Close this PR and create new one.

**Pull request closed without merging**